ansible/configs/ansible-automation-platform/roles/automation_controller_issue_cert_cfg/tasks/main.yml

```diff
@@ -0,0 +1,106 @@
+---
+- name: SSL cert block
+  block:
+    - name: make sure Ansible Tower is online before changing tower base URL
+      service:
+        name: automation-controller.service
+        state: started
+      register: start_tower
+      until: start_tower is not failed
+      retries: 5
+    
+    - name: Enable EPEL
+      dnf:
+        name: "https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm"
+        state: present
+        disable_gpg_check: true
+
+    # directions found here https://certbot.eff.org/lets-encrypt/centosrhel8-other
+    - name: install certbot if not already installed
+      dnf:
+        name: certbot
+        state: present
+        disable_gpg_check: true
+
+    - &tower-pinger-block
+      block:
+        - name: check Tower status
+          shell: "curl --user 'admin:{{ hostvars[groups['bastions'][0]]['tower_admin_password'] }}' -vkL -XGET https://localhost/api/v2/ping/"
+          register: check2
+        - name: Display /api/v2/ping results (stdout)
+          debug:
+            msg: '{{ check2.stdout }}'
+        - name: Display /api/v2/ping results (stderr)
+          debug:
+            msg: '{{ check2.stderr }}'
+
+    - name: change ansible tower base URL
+      block:
+        - &tower-baseurl-task
+          awx.awx.tower_settings:
+            name: TOWER_URL_BASE
+            value: "https://{{ ansible_host.split('.')[0] | lower }}.{{ subdomain_base }}"
+            tower_verify_ssl: false
+            tower_host: https://localhost/
+            tower_username: admin
+            tower_password: "{{ hostvars[groups['bastions'][0]]['tower_admin_password'] }}"
+          register: change_base_url
+          until: change_base_url is not failed
+          # tower may not be fully up yet, wait for it gratuitously
+          retries: 12
+          delay: 5
+      rescue:
+        - name: Try restarting tower
+          service:
+            name: automation-controller.service
+            state: restarted
+
+        - *tower-pinger-block
+
+        - <<: *tower-baseurl-task
+          name: Run failed task again
+
+    # https://docs.ansible.com/ansible-tower/latest/html/administration/init_script.html
+    - name: make sure Ansible Tower is stopped
+      service:
+        name: automation-controller.service
+        state: stopped
+      register: stop_tower
+      until: stop_tower is not failed
+      retries: 5
+
+    # If this fails check out status of certbot: https://letsencrypt.status.io/
+    - name: ISSUE CERT
+      shell: certbot certonly --no-bootstrap --standalone -d {{ ansible_host.split(".")[0] | lower }}.{{ subdomain_base }} --email ansible-network@redhat.com --noninteractive --agree-tos
```

Comment:
you're not using any feature of the `shell` module, `command` should be enough

Please wrap the line

```yaml
command: >-
  certbot certonly --no-bootstrap
  --standalone ...
```